Remotion/Web/Development.WebTesting/WebTestSetUpFixtureHelper.cs

```diff
@@ -104,10 +106,12 @@ public string LogDirectory
     public void OnSetUp ()
     {
       HostWebApplication();
+      if (!_testSiteStartupCheckEnabled)
+        return;
 
       try
       {
-        VerifyWebApplicationStarted(_webApplicationRoot, _verifyWebApplicationStartedTimeout);
+        VerifyWebApplicationStarted(_testSiteStartupCheckUrl, _verifyWebApplicationStartedTimeout);
       }
       catch
       {
```

Comment:
Hmm...I'm not convinced the boolean should be an exit conditon instead of an IF block. Reasoning: OnSetUp in the FixtureHelper is more than just start and optionally verify. Or rather, it's coincidental that right now, that's the only stuff we do. So, if we ever need to do more, we need to recogized the IF as actually being a condtional block of code, determine how far it goes and then decide how to refactor / insert new code. If, instead, we make this into a block right now, it's immediately clear.